openam-ui/openam-ui-ria/pom.xml

```diff
@@ -285,8 +285,8 @@
                         <configuration>
                             <nodeVersion>v4.4.5</nodeVersion>
                             <npmVersion>3.9.3</npmVersion>
-                            <downloadRoot>http://maven.forgerock.org/repo/forgerock-third-party-virtual/</downloadRoot>
-                            <npmDownloadRoot>http://maven.forgerock.org/repo/api/npm/npm-virtual/npm/-/</npmDownloadRoot>
+                            <downloadRoot>https://nodejs.org/dist/</downloadRoot>
```

Comment:
If it's possible, it might be a good idea for us to setup a virtual package for NPM itself, like FR did.

Reply:
Yeah that seems like a good idea though I'm not entirely sure how they did it. Judging from the URL I would say they just uploaded it to a Maven or Generic repo. Though the Maven plugin probably expects a specific directory layout. I guess that would entail downloading the stuff under https://nodejs.org/dist/v4.4.5/ and uploading it to Artifactory. Those artifacts are somewhat large. How much space do we have in our Artifactory instance?

Reply:
Looks like they give us 8 GB but I think we can request more for OSS if needed:
![image](https://user-images.githubusercontent.com/27592108/41075652-a7865160-69db-11e8-9f6b-e23653e97163.png)

Reply:
I see your point, though, Jasper. that could end up being a lot of npm artifacts. usually JFrog caches them on demand though, so at least it's not mirroring the entirety of npm.